scope.go

```diff
@@ -41,6 +41,9 @@ type scope struct {
 	remoteAddr string
 	localAddr  string
 
+	// is true when KillQuery has been called
+	cancelled bool
```

Comment:
Rename to `canceled` - see https://www.grammarly.com/blog/canceled-vs-cancelled/ :)